Frontend/default.nix

```diff
@@ -1,28 +1,69 @@
 { inputs, ... }:
 {
   imports = [
+    ./node
+    ./nix/watch.nix
+    ./nix/bundle.nix
   ];
-  perSystem = { config, self', pkgs, lib, ... }:
-    let easy-ps = import inputs.easy-purescript-nix { inherit pkgs; };
-    in {
+
+  perSystem = { config, self', lib, system, ... }:
+    let
+      pkgs = import inputs.nixpkgs {
+        inherit system;
+        overlays = [
+          inputs.purifix.overlay
+        ];
+      };
+
+      localPackages = pkgs.purifix {
+        src = ./.;
+      };
+
+      # Wrapper for typescript language server to make it work with neovim(and other editors) by default
+      # Ensure that typescript is installed and the language server knows where tsserver lives
+      # The --tsserver-path argument being hardcoded instead of sent in the initialization
+      # both simplifies the process and ensures that everyone is using the same tsserver when
+      # the editor is running in the development shell
+      typescript-language-server = pkgs.symlinkJoin {
+        name = "typescript-language-server";
+        nativeBuildInputs = [ pkgs.makeWrapper ];
+        paths = [
+          pkgs.nodePackages.typescript
+          pkgs.nodePackages.typescript-language-server
+        ];
+        postBuild = ''
+          wrapProgram $out/bin/typescript-language-server \
+            --add-flags "--tsserver-path $out/bin/tsserver"
+        '';
+      };
```

Comment:
Move this to `./nix/tsserver.nix`, so that we can keep `Frontend/default.nix` as simpler/smaller as possible.